### DISCUSSION TITLE
increasing wind penetration by ANDES

### DISCUSSION THREAD

**Discussion by Mishal2024** [CURENT/andes#460] [Ideas]
Hi Hantao, 

Thank you for your interesting

I have my own Saudi 222 Buses model case and I already done from convert Raw & Dry files to Excel file in ANDES. 

My work is focusing on increase wind penetration and checked the Impact of increasing on the frequency.

How can I start from the base case, and add 30% renewable each time to see what’s the highest penetration level for the model case with no running issue by ANDES.

**Comment by cuihantao** [comment]
Hi Mishal2024,

Thank you.

Instead of adding 30% each time, you can replace one generator by a renewable at a time and see if power flow will solve, if the system can be properly initialized, and if the system will be stable following the disturbance. Refer to the test cases in `andes/cases/ieee14/ieee14_wt3.xlsx` for generic parameters for wind.

**Comment by Mishal2024** [comment]
Thank you Dr.Cuihantao for support

I appreciate your time
